app/lib/features/chat/pages/room_profile_page.dart

```diff
@@ -0,0 +1,146 @@
+import 'package:acter/common/providers/common_providers.dart';
+import 'package:acter/common/providers/space_providers.dart';
+import 'package:acter/common/themes/app_theme.dart';
+import 'package:acter/common/utils/routes.dart';
+import 'package:acter/common/widgets/spaces/space_parent_badge.dart';
+import 'package:acter/features/chat/providers/chat_providers.dart';
+import 'package:acter/features/chat/widgets/member_list.dart';
+import 'package:acter_avatar/acter_avatar.dart';
+import 'package:atlas_icons/atlas_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:go_router/go_router.dart';
+
+class RoomProfilePage extends ConsumerWidget {
+  const RoomProfilePage({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final room = ref.watch(currentConvoProvider)!;
+    final members =
+        ref.watch(chatMembersProvider(room.getRoomIdStr())).requireValue;
```

Comment:
we can't be sure this has loaded yet though ... a large room, the user might click on the chat and the profile long before loading of the members (from remote) has succeeded. That should not block the user from going here and using that page. Let's refactor that to the a `members.when` and show a loading animation please.

Reply:
Here it only shows member list count. Anyways I converted it into ```AsyncValue```.